### PR TITLE
Add rel attribute to links with target _blank

### PR DESCRIPTION
**Pull request by anselmbradford**
Adds `rel=\"noopener noreferrer\"` to links that target `_blank`, which checkmarx complains about.

### CODE REVIEW

File: retirement_api/utils/ss_utilities.py [hunk 1]
```diff
@@ -11,22 +11,25 @@
...
 TOO_OLD = """\
 <span class="h4">Sorry, our tool cannot provide an estimate because \
 your birthdate, {0}, means you are older than 70 and may already be receiving \
 benefits.</span> To check your benefits based on your actual \
 earnings record, contact the Social Security Administration or \
-open a <a href="http://www.socialsecurity.gov/myaccount/" target="_blank">\
-<em>my</em> Social Security</a> account.
+open a <a href="http://www.socialsecurity.gov/myaccount/" \
+target="_blank" rel="noopener noreferrer">\
+<em>my</em> Social Security</a> account.\
```
Comment (anselmbradford) [inline retirement_api/utils/ss_utilities.py]: I added a trailing `\` here as all other blocks of text in this file had it for the last line. Is this correct?

File: retirement_api/utils/ss_utilities.py [hunk 2]
```diff
@@ -11,22 +11,25 @@
 if you are under 22 years of age.</span> Please visit the \
 Social Security Administration's \
 <a href="http://www.ssa.gov/people/youngpeople/" \
-target="blank">advice page</a> for students and younger workers.\
+target="_blank" rel="noopener noreferrer">advice page</a> \
```
Comment (anselmbradford) [inline retirement_api/utils/ss_utilities.py]: Added `_blank` here as `blank` appeared to be a typo.

**Comment by anselmbradford** [comment]
cc @lfatty

File: retirement_api/utils/ss_utilities.py (hunk 1 above)
Comment (higs4281) [reply retirement_api/utils/ss_utilities.py]: Final backslash is no-op. At least our no-ops are consistent now!

**Pull request merged**